llvm/utils/git/github-automation.py

```diff
@@ -65,6 +66,30 @@ def run(self) -> bool:
         return False
 
 
+class PRSubscriber:
+    @property
+    def team_name(self) -> str:
+        return self._team_name
+
+    def __init__(self, token: str, repo: str, pr_number: int, label_name : str):
+        self.repo = github.Github(token).get_repo(repo)
+        self.org = github.Github(token).get_organization(self.repo.organization.login)
+        self.pr = self.repo.get_issue(pr_number).as_pull_request()
+        self._team_name = "pr-subscribers-{}".format(label_name).lower()
+
+    def run(self) -> bool:
+        for team in self.org.get_teams():
+            if self.team_name != team.name.lower():
+                continue
+            try:
+                patch = requests.get(self.pr.diff_url).text
```

Comment:
Can this be arbitrarily large? Shall we limit the size here with a subset? 
I don't know the limit for a comment size with GitHub API but I would be afraid of the comment posting failing because of that.

Reply:
It seems like the comments are limited to 65536 characters.  We can try to put some kind of limit on this.